decisions/mnm_hermetics_decisions.txt

```diff
@@ -348,75 +349,97 @@ targeted_decisions = {
 		ai_check_interval = 12
```

Comment:
Since it has a long cooldown, I would make the check interval half of the cooldown.
```suggestion
		ai_check_interval = 180
```